### DISCUSSION TITLE
Clarity on Pricing

### DISCUSSION THREAD

**Discussion by allindeveloper** [vercel/vercel#4600] [Help] [answered]
I would like some clarity on my billings, so as soon as I put created the team around May 26 and put in my debit card.. $20 was debited from me…

And now I am seeing an upcoming invoice amount of $20 which is due on June 9, 2020, and how about the 14 day free period.

I don’t understand.

Please I need some clarity.

**Comment by mcsdevv** [comment, marked as answer]
Hello @allindeveloper, please contact Vercel Support at vercel.com/support/request so that we can investigate this for you.

**Reply by allindeveloper** [reply]
Thank You @mcsdevv  I have already contacted support and gotten it resolved.

**Reply by mcsdevv** [reply]
Awesome, glad to hear it!
